test/schemes/fluid/surface_tension.jl

```diff
@@ -0,0 +1,88 @@
+
+@testset verbose=true "surface tension" begin
+    @testset verbose=true "cohesion_force_akinci" begin
+        surface_tension = TrixiParticles.SurfaceTensionAkinci(surface_tension_coefficient=1.0)
+        support_radius = 1.0
+        m_b = 1.0
+        pos_diff = [1.0, 1.0]
+
+        test_distance = 0.1
+        val = TrixiParticles.cohesion_force_akinci(surface_tension, support_radius, m_b,
+                                                   pos_diff, test_distance) * test_distance
+        @test isapprox(val[1], 0.1443038770421044, atol=6e-15)
+        @test isapprox(val[2], 0.1443038770421044, atol=6e-15)
+
+        # maximum repulsion force
```

Comment:
```suggestion
        # Maximum repulsion force
```
Please also check the other comments.